### DISCUSSION TITLE
Banana Pi M2?

### DISCUSSION THREAD

**Discussion by PaulWieland** [mikebrady/shairport-sync#1692] [General]
Since all models of raspberry pi seem to be no long available to mere mortals, I’m looking for an alternative in order to run Shairport sync / airplay 2. 

Has anyone tried a banana pi m2 zero?  Alternative hardware suggestions welcome. 

I was planning to use a PCM5102A DAC.

**Comment by mikebrady** [comment]
Hmm. My experience with some of these devices has been poor. Also, [this](https://arstechnica.com/gadgets/2023/06/ceo-raspberry-pi-inventory-to-hit-1m-units-monthly-starting-in-july/). Might be worth holding on...
